app/Http/Controllers/Admin/AdminAuthController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Auth;
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\Http\Requests\Admin\AdminLoginRequest;
+
+class AdminAuthController extends Controller
```

Comment:
class name should be `AuthController` because we already have `Admin` namespace. 
Extend from `AdminBaseController`